### ISSUE TITLE
Segmentation fault on serializing empty dataclass

### ISSUE THREAD

**Issue by barsi** [ijl/orjson#115]
**OS** Ubuntu 20.04

**Python** 3.8.2

**orjson** 3.2.1

**Example:**

```python
from dataclasses import dataclass, asdict
import orjson

@dataclass
class A:
  pass

print(asdict(A())) # -> {}
print(orjson.dumps(A())) # -> segmentation fault
```

**Comment by ijl** [comment]
Fixed in 3.2.2.

**Comment by mashbourne** [comment]
Is there an ETA for release of 3.2.2?
